### ISSUE TITLE
Allow for trim_blocks and lstrip_blocks in jinja2 templates

### ISSUE THREAD

**Issue by metacyclic** [Sceptre/sceptre#952]
This can be accomplished with the judicious application of `-` chars in the jinja2 templates, but it would be nice to pass this in as a config variable.  This could be accomplished a couple of ways:

1) in the vars.yaml file
```
template_path: 'my.yaml.j2'
template_options: 'trim_blocks: True, lstrip_blocks: True'
```

2) via a magic comment, as Ansible does.  In the template file, first line
```
#jinja2: lstrip_blocks: True, trim_blocks: True
```

This would allow for faster template development, and faster adoption, IMO.  I believe that this could be added [here](https://github.com/Sceptre/sceptre/blob/03806d1b407d293237fff7a6f3533a63d553205d/sceptre/stack.py#L270), but not entirely sure.

**Comment by zaro0508** [comment]
I like option 1 however I think the setting would be better if it's in the StackGroup config so we don't have to set it in every file?

in config.yaml
```
jinja2_environment:
  trim_blocks: {{ var.trim_blocks | default("false") }}
  lstrip_blocks: {{ var.lstrip_blocks | default("false") }} 
```

from terminal:
```
sceptre --var "jinja2_environment.trim_blocks=true" --var "jinja2_environment.lstrip_blocks=true" launch dev
```

**Comment by metacyclic** [comment]
@zaro0508 I like it!  Defining it globally makes a lot more sense.

**Comment by zaro0508** [comment]
fixed in PR #1079
